website/source/guides/operations/performance-nodes.html.md

```diff
@@ -20,7 +20,7 @@ _standby_ nodes and simply forward requests to the _active_ node.
 
 ![HA Architecture](/assets/images/vault-ha-consul-3.png)
 
-If you are running **_Vault Enterprise_ 0.11** or later, those standby nodes can
+If you are running **_Vault Enterprise_ 0.11** or later with the Consul storage backend, those standby nodes can
```

Comment:
Can you fix the line break of this line and the resulting paragraph?

Reply:
Edited and committed.